api/controllers/user.go

```diff
@@ -85,6 +85,17 @@ func (uc *UserController) Login(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(201)
 }
 
+// @Summary Update user information
+// @Description Updates the details of an existing user based on the provided user ID and payload
+// @Tags User
+// @Produce json
+// @Param userId path string true "User ID"
+// @Param user body dto.UserUpdateDto true "User update payload"
+// @Success 200 {object} dto.UserUpdateDto
+// @Failure 400
+// @Failure 404
+// @Failure 500
+// @Router /users/{userId} [put]
```

Comment:
Olá, João Marcelo! Obrigado pelas sua contribuições. Sempre bom receber novos colaboradores. Sobre sua task, notei que não estou conseguindo inserir os dados para atualizar. Você conseguiu inserir ID e Body diretamente pela página da documentação ao clicar na opção "try it out"?